### ISSUE TITLE
ImportError: libcupti.so.11.7: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by silvacarl2** [sanchit-gandhi/whisper-jax#108]
after doing this:

pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

we now get this:

ImportError: libcupti.so.11.7: cannot open shared object file: No such file or directory

nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Tue_May__3_18:49:52_PDT_2022
Cuda compilation tools, release 11.7, V11.7.64
Build cuda_11.7.r11.7/compiler.31294372_0

any ideas what is missing?

we are testing this on WSL2/Ubuntu 20.04

**Comment by themanyone** [comment]
I got cuda-cupti package for Fedora. It came with the cuda 11-7 install from the nvidia website.

**Comment by silvacarl2** [comment]
yeah i thought so too and i found it but no matter i do, it doesnt seem to see it

**Comment by themanyone** [comment]
Oh, I also had to set some environment in .bashrc to get it to recognise.

```
export CUDA_HOME=/usr/local/cuda
export LIBRARY_PATH="${CUDA_HOME}/lib64"
export C_INCLUDE_PATH="${CUDA_HOME}/include"
```

**Comment by silvacarl2** [comment]
got it, that works.  8-)

**Comment by alpcansoydas** [comment]
Just install pip install transformers[torch]

**Comment by alpcansoydas** [comment]
I downgraded torch to 2.0.0 and its fixed

**Comment by ChauhanT** [comment]
Can confirm: downgrading to 2.0.0 solves the issue.
